### PR TITLE
Fixes #1613 | Adds a better treatment to i18n when one item is selected inside ManagementToolbar

### LINKED ISSUE #1613: Management toolbar needs to contextualize message when one item is selected
Since one item is selected, the info message should use a singular (not plural) noun.

#### Expected result:

> 1 of 1 item selected. or 1 item selected.

#### Actual result:

> All 1 items selected. displays. This is grammatically incorrect.

### PR DESCRIPTION
**Pull request by diegonvs**

### CODE REVIEW

**Comment by carloslancha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by carloslancha** [comment]
Hey @diegonvs reviewing the docs and talking with @victorvalle looks like this is not complete correct.

https://docs.google.com/document/d/1Aov9rThS1bcVscmS36DyCbpHdrM0OcEJZhA4w4_kHR4/edit?ts=5c77b2a8#heading=h.r0dec54wvvd4

When all the elements are selected the correct text should be just `All Selected`.

Also when all elements are selected the `Select All` button shouldn't appear (what happens after this pr).

Could you please take a look at the doc and resend with required changes? Thx!!

**Comment by diegonvs** [comment]
Sure!

**Comment by matuzalemsteles** [comment]
hey @carloslancha the logic of whether or not to show the `Select All` button was not removed? You introduced this in #1511 to be able to make it more flexible and introduce this logic into Liferay.

**Comment by diegonvs** [comment]
Hey @carloslancha, about `All selected` message, I followed the message according ticket description. See https://issues.liferay.com/browse/LPS-90796.

**Comment by carloslancha** [comment]
@matuzalemsteles you're totally right, I just saw the demo and forgot about all that stuff. @diegonvs pelease update the demo so if all items are selected select all button doesn't appear to be consistent with lexicon.

@diegonvs about "All selected", Q.A. reported the wrong behavior but decision must be taken by lexicon, so let's go with "All selected" ;)

File: packages/clay-management-toolbar/demos/index.html [hunk 1]
```diff
@@ -351,6 +351,48 @@ <h2>With results bar and filter labels</h2>
...
+			checkboxStatus: 'checked',
+			selectable: true,
+			selectedItems: 1,
+			showInfoButton: true,
+			showSelectAllButton: true,
+			spritemap: spritemap,
+			supportsBulkActions: true,
+			totalItems: 10,
+		},
+		'#active-checked-select-all-block');
```
Comment (matuzalemsteles) [inline packages/clay-management-toolbar/demos/index.html]: Can you separate these new examples in blocks? the same way we do with existing ones.

File: packages/clay-management-toolbar/src/ClayManagementToolbar.soy [hunk 2]
```diff
@@ -253,21 +253,18 @@
 				</li>
 				<li class="nav-item">
 					<span class="navbar-text">
-						{if $supportsBulkActions and $selectedItems == $totalItems}
-							{msg desc="Says that all possible items are selected when bulk actions are available"}
-								all-{$totalItems}-items-selected
-							{/msg}
-						{else}
-							{msg desc="Says how many items of the total are selected"}
-								{$selectedItems}-of-{$totalItems}
```
Comment (matuzalemsteles) [inline packages/clay-management-toolbar/src/ClayManagementToolbar.soy]: I think we should keep this message still, but do not show it when the number of items selected and the total items are the same.

File: packages/clay-management-toolbar/src/ClayManagementToolbar.soy [hunk 3]
```diff
@@ -253,21 +253,18 @@
 				</li>
 				<li class="nav-item">
 					<span class="navbar-text">
-						{if $supportsBulkActions and $selectedItems == $totalItems}
-							{msg desc="Says that all possible items are selected when bulk actions are available"}
-								all-{$totalItems}-items-selected
```
Comment (matuzalemsteles) [inline packages/clay-management-toolbar/src/ClayManagementToolbar.soy]: I think we should just change this to:

```
all-selected
```

**Comment by carloslancha** [comment]
On monday I'll review this with @victorvalle, looks like definition could not be correct. I'll keep you updaged guys, sorry. @matuzalemsteles @diegonvs

File: packages/clay-isomorphic/fixtures/expected/ClayManagementToolbar.active [hunk 4]
```diff
@@ -1 +0,0 @@
...="true"class="dropdown-toggle btn nav-link nav-link-monospaced btn-unstyled"data-onclick="toggle"ref="triggerButton"title="actions"type="button"><svg class="lexicon-icon lexicon-icon-ellipsis-v"focusable="false"role="presentation"><use xlink:href="spritemap.svg#ellipsis-v" /></svg></button><div class="dropdown-menu" ref="menu"><ul class="list-unstyled"><li data-onclick="null" tabindex="-1"><button class="btn dropdown-item btn-sm btn-link"aria-label="action 1"type="button">action 1</button></li><li data-onclick="null" tabindex="-1"><button class="btn dropdown-item btn-sm btn-link"aria-label="action 2"type="button">action 2</button></li><li data-onclick="null" tabindex="-1"><button class="btn dropdown-item btn-sm btn-link"type="button"></button></li><li data-onclick="null" tabindex="-1"><button class="btn dropdown-item btn-sm btn-link"type="button"></button></li><li data-onclick="null" tabindex="-1"><button class="btn dropdown-item btn-sm btn-link"aria-label="action 5"type="button">action 5</button></li><li data-onclick="null" tabindex="-1"><button class="btn dropdown-item btn-sm btn-link"aria-label="action 6"type="button">action 6</button></li></ul></div></div></li></ul></div></nav>
```
[the remark this answers is not in the document]
Comment (diegonvs) [reply packages/clay-isomorphic/fixtures/expected/ClayManagementToolbar.active]: It was removed by gradle task

File: package.json [hunk 5]
```diff
@@ -6,7 +6,8 @@
     "checkFormat": "npm run prettier -- --list-different",
     "coveralls": "node node_modules/coveralls/bin/coveralls.js < coverage/lcov.info",
     "format": "npm run prettier -- --write",
-    "jest": "cross-env NODE_ENV=test jest",
+    "jest": "rm -rf clayui.com/.cache && cross-env NODE_ENV=test jest",
```
[the remark this answers is not in the document]
Comment (diegonvs) [reply package.json]: Makes sense!

File: packages/clay-management-toolbar/demos/index.html (hunk 1 above)
Comment (diegonvs) [reply packages/clay-management-toolbar/demos/index.html]: Yes!

File: packages/clay-management-toolbar/src/ClayManagementToolbar.soy (hunk 2 above)
Comment (diegonvs) [reply packages/clay-management-toolbar/src/ClayManagementToolbar.soy]: Was defined in https://issues.liferay.com/browse/LPS-90796 to use `All Selected` even if the number of selected items is 1.

**Comment by matuzalemsteles** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by carloslancha** [comment]
@victorvalle could you please review this? After talking with @wincent and @jbalsas it looks like the text we decided could not be totally correct.

**Comment by victorvalle** [comment]
We have decided to remove the word "items" from the management toolbar as it doesn't add any value in the text.

**Comment by diegonvs** [comment]
So, the new behaviour when selecting, for example, one of one value is: 
- 1 of 1 ~items~ selected(without the word items), or,
- All selected, or
- 1 of 1

and, one of ten value is:

- 1 of 10 items selected, or
- 1 of 10 selected

@victorvalle

**Comment by victorvalle** [comment]
Hi @diegonvs the correct way is:
- 1 of 1 selected
- 1 of 10 selected

**Comment by carloslancha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by carloslancha** [comment]
Resent https://github.com/liferay/clay/pull/1718

**Pull request closed without merging**